RELEASE_NOTES.md

```diff
@@ -1,5 +1,16 @@
 # Assembly Homology Service release notes
 
+## 0.1.5
+
+* The MongoDB clients have been updated to the most recent version.
+* Added the ``mongo-retrywrites`` configuration setting in ``deploy.cfg``, defaulting to
+  ``false``.
+
+## 0.1.4
+
+* Switched the build system from Ant to Gradle. As such, all build artifacts, including the
+  `assembly_homology` script, are now found in the `build` directory.
+
```

Comment:
If 0.1.4 has never been released, we should just remove the 0.1.5 release and merge the notes

Reply:
That's what I thought. I should have just added it to the 0.1.4 notes.

Reply:
👍